lib/__tests__/date-utils.test.ts

```diff
@@ -0,0 +1,53 @@
+import { getCurrentLongDate, getCurrentMonthAndYear } from "../date-utils";
+
+describe("getCurrentLongDate", () => {
+  let dateSpy: jest.SpyInstance;
```

Comment:
No need to use this, Mas. I have mine implemented on #394 using Jest utilities.

Reply:
Better in #394, for now I already skip the test